docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -0,0 +1,64 @@
+= Parameters
+
+The parent key for all of the following parameters is `metallb`.
+
+
+== `namespace`
+
+[horizontal]
+type:: string
+default:: `syn-metallb`
+
+
+
+== `speaker.secretkey`
+
+[horizontal]
+type:: string
+default:: Vault reference
+
+A binary secret key encoded base64.
+It can be generated using the command `openssl rand -base64 128`.
+
+== `addresses`
+
+[horizontal]
+type:: list
+default:: null
+
+A list of IPs (for example `10.128.1.200/32`) used as transferable virtual IPs.
+
+
+== `config`
+
+[horizontal]
+type:: dictionary
+default:: null
+
+A custom config can be provided.
+This overrides the automatic config generation.
+
+NOTE: The passed configuration is not going to be validated by the component.
```

Comment:
Probably makes sense to link to the MetalLB docs here. Or at least mention that without an address pool called "default", each LB service must have the annotation specifying which pool the IP should be allocated from.

Reply:
Added